uber/config.py

```diff
@@ -73,9 +73,19 @@ def make_enum(self, enum_name, section, prices=False):
                 lookup[val] = desc
 
         enum_name = enum_name.upper()
-        setattr(self, enum_name + '_OPTS', opts)
-        setattr(self, enum_name + '_VARS', varnames)
-        setattr(self, enum_name + ('' if enum_name.endswith('S') else 'S'), lookup)
+        # When loading plugin's configs, we want to make sure we don't override existing enums
+        if hasattr(self, enum_name + '_OPTS'):
```

Comment:
I think I like that idea over trying to intercept the post-processed like in this PR.  I guess that's just functionality to merge two ConfigObj's together, which I think we're vaguely already doing somewhere between development.ini, development-defaults.ini/etc

That actually solves another issue I was having where core uber couldn't easily use config data from a plugin if it was present.

I guess this would look like?
1) read in all different .ini's into configobj's
2) merge all of that together
3) post-process it with the config.py in core and the plugins